nixos/modules/services/misc/spice-autorandr.nix

```diff
@@ -0,0 +1,27 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
We try to reduce the amount of `with lib;` so new modules should not introduce this.